### ISSUE TITLE
Problem with plasTeX on python3.7 macOS 

### ISSUE THREAD

**Issue by samtsevich** [plastex/plastex#103]
Hey,

Just installed plasTeX on my mac and tried to run unittests. And half of them failed (see report.txt file).
Any suggestions what is wrong and how to fix it.

Cheers
[report.txt](https://github.com/tiarno/plastex/files/2254327/report.txt)

**Comment by kesmit13** [comment]
plasTeX still only runs on Python 2.7.  There have been attempts to create a Python 3 version, but they haven't been made production yet.

**Comment by PatrickMassot** [comment]
As far as I know the python3 branch is usable, and used. For instance it produces https://stacks.math.columbia.edu/ (more than 6000 pages of heavy maths converted to HTML by plasTeX). @samtsevich could you tell me more? How did you install plasTeX? If you cloned it from here, which branch are you running?

**Comment by samtsevich** [comment]
Here what I have, when I tried to use plasTeX compiled with python2.7

```
plastex    manual_english.tex 
Traceback (most recent call last):
  File "/usr/local/bin/plastex", line 4, in <module>
    __import__('pkg_resources').run_script('plasTeX==1.0', 'plastex')
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 657, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1437, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python2.7/site-packages/plasTeX-1.0-py2.7.egg/EGG-INFO/scripts/plastex", line 5, in <module>
    import plasTeX
  File "/usr/local/lib/python2.7/site-packages/plasTeX-1.0-py2.7.egg/plasTeX/__init__.py", line 5, in <module>
    from plasTeX import Logging, encoding
  File "/usr/local/lib/python2.7/site-packages/plasTeX-1.0-py2.7.egg/plasTeX/Logging.py", line 6, in <module>
    from plasTeX.Config import config as _config
  File "/usr/local/lib/python2.7/site-packages/plasTeX-1.0-py2.7.egg/plasTeX/Config.py", line 4, in <module>
    from plasTeX.ConfigManager import *
  File "/usr/local/lib/python2.7/site-packages/plasTeX-1.0-py2.7.egg/plasTeX/ConfigManager/__init__.py", line 18, in <module>
    import sys, string, re, urllib.request, copy, types, os
ImportError: No module named request
```

@PatrickMassot Any ideas what can be wrong?

**Comment by PatrickMassot** [comment]
This error strongly suggests you are trying to use the python3 branch with python 2.7, which is hopeless. How did you install plasTeX?

**Comment by PatrickMassot** [comment]
Fixed in #118.
